routers/api/packages/api.go

```diff
@@ -265,6 +266,20 @@ func Routes() *web.Route {
 				r.Delete("/yank", rubygems.DeletePackage)
 			}, reqPackageAccess(perm.AccessModeWrite))
 		})
+		r.Group("/vagrant", func() {
+			r.Group("/authenticate", func() {
+				// r.Post("", vagrant.Authenticate)
```

Comment:
:eyes:

Reply:
I kept it as reminder if we want to support the CLI login in future. But for the current (working) implementation it's not needed and can be removed if you'd like to.

Reply:
```suggestion
```